README.md

```diff
@@ -21,7 +21,7 @@
   - [Install](#install)
```

Comment:
**[OPTIONAL]**

- [ ] It would be very nice if you make all your commit messages capitalized to make your pull request professional :+1: